<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateGuideLines</class>
 <widget class="QDialog" name="CreateGuideLines">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>234</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Guide Lines</string>
  </property>
  <widget class="QComboBox" name="AlignmentCB">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="GuideLinesCB">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="SelecetAlignmentL">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Alignment:</string>
   </property>
  </widget>
  <widget class="QLabel" name="SelectGuideLinesL">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Guide Lines Group:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="CreateGroupChB">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Create New Group</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LeftLengthLE">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>20</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="RightLengthLE">
   <property name="geometry">
    <rect>
     <x>302</x>
     <y>110</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>20</string>
   </property>
  </widget>
  <widget class="QLabel" name="LeftLengthL">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>112</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Left Line Length:</string>
   </property>
  </widget>
  <widget class="QLabel" name="RightLengthL">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>112</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Right Line Length:</string>
   </property>
  </widget>
  <widget class="QLabel" name="IncrementL">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Increment Along Aligment:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="IncrementLE">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>150</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CreateB">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelB">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
